src/components/bookReact.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import Books from './books';
+import AddNewBook from './addBook';
+
+const DisplayBook = () => {
+  const [book] = useState(Books);
+  const ulMenu = {
+    display: 'flex',
+    listStyle: 'none',
+    gap: '20px',
+    alignItems: 'center',
+  };
+  const deleteButton = {
+    border: '2px solid teal',
+    padding: '5px',
+    cursor: 'pointer',
+  };
+  const wrapper = {
+    border: '2px solid grey',
+    marginBottom: '10px',
+  };
+  return (
+    <div>
+      <section>
+        {book.map((book) => (
+          <div key={book.id} style={wrapper}>
+            <p>
+              Title:
+              {book.title}
+            </p>
+            <p>
+              Author:
+              {book.author}
+            </p>
+            <ul style={ulMenu}>
+              <li>Comments</li>
+              <li style={deleteButton}>Remove</li>
+              <li>Edit</li>
+            </ul>
+          </div>
+        ))}
```

Comment:
- [OPTIONAL] It would be better to create a Button component and use them in your `book` iteration.
- You can use props in functional components too, like in the example.

**Passing props to component**
![button-component](https://user-images.githubusercontent.com/79876959/144972077-3360d365-06a3-4672-b272-04dfb7e07630.PNG)

**Button component** (You can use destructuring here because book is an object, for other types of variables is a little bit different.)
![button-component-2](https://user-images.githubusercontent.com/79876959/144971942-7c9c2bd9-da4c-41ef-946e-942639973412.PNG)

Reply:
Thanks a lot for your comments, I didn't know I can use props in functional components, thanks @ArthurGC 🤓